services/impl/src/main/java/org/exoplatform/automatic/translation/impl/AutomaticTranslationServiceImpl.java

```diff
@@ -62,16 +66,15 @@ public void addConnector(AutomaticTranslationComponentPlugin translationConnecto
   public String getActiveConnector() {
     SettingValue<?> value = settingService.get(Context.GLOBAL, Scope.GLOBAL, AUTOMATIC_TRANSLATION_ACTIVE_CONNECTOR);
 
-
-    return value != null && value.getValue()!=null && !value.getValue().toString().isEmpty() && isActiveFeature() ?
-           value.getValue().toString() : null;
+    return value != null && value.getValue() != null && !value.getValue().toString().isEmpty()
+        && isActiveFeature() ? value.getValue().toString() : null;
   }
 
   private boolean isActiveFeature() {
-    String enable = System.getProperty("exo.feature."+FEATURE_NAME+".enabled");
-    return (enable!=null && !enable.isBlank() && enable.equals("true"));
-    //after feature finished, return this :
-    //return exoFeatureService.isActiveFeature(FEATURE_NAME);
+    String enable = System.getProperty("exo.feature." + FEATURE_NAME + ".enabled");
```

Comment:
I need to have a default value to false because the feature is not finished.
As far as I know, with feature service, the default value is true.
So I'll use feature service when the feature is activated by default

Reply:
To change the default value, you can proceed as we do in other Feature flags in fact, see this example :
https://github.com/Meeds-io/social/blob/e6f2fd7f4b02bb2d4e0dd7890b1940dca8774bcd/extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/core-configuration.xml#L719-L729

Reply:
ok, done